### PR TITLE
Feature/update exporter for results narrative

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: src/TransformCore/PHE/HayApi/NarrativeBundle/Narrative/Results/ResultsNarrativeBuilder.php [hunk 1]
```diff
@@ -31,6 +34,8 @@
 class ResultsNarrativeBuilder extends NarrativeBuilderAbstract
 {
 
+    const SECTION_NAME = "results";
```
Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/NarrativeBundle/Narrative/Results/ResultsNarrativeBuilder.php]: This constant also refers to the translation domain, should I rather rename it to `TRANSLATION_DOMAIN` or similar, section name makes sense but it serves 2 purposes

Thoughts?

Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/NarrativeBundle/Narrative/Results/ResultsNarrativeBuilder.php]: Currently it reads `ResultsNarrativeBuilder::SECTION_NAME`, kind of makes sense. 

Would `ResultsNarrativeBuilder:: TRANSLATION_DOMAIN ` make more sense?

File: src/TransformCore/PHE/ExporterBundle/Model/Record.php [hunk 2]
```diff
@@ -77,9 +79,16 @@ public function __construct(Questionnaire $questionnaire)
...
+            ResultsNarrativeHelper::extractResultsNarrative($questionnaire, $translator)
+        );
+
+        $input = array_merge(
+            $input,
             $this->extractRagScore($questionnaire)
         );
 
+
+
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Model/Record.php]: 3 blank lines?

File: src/TransformCore/PHE/HayApi/NarrativeBundle/Narrative/Results/ResultsNarrativeBuilder.php (hunk 1 above)
Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/NarrativeBundle/Narrative/Results/ResultsNarrativeBuilder.php]: Yes and no, I think for someone not knowing what the code does, `SECTION_NAME` is a bit more descriptive whilst not 100% accurate, am probably overthinking this :D

File: src/TransformCore/PHE/ExporterBundle/Model/Record.php (hunk 2 above)
Comment (brettminnie) [inline src/TransformCore/PHE/ExporterBundle/Model/Record.php]: OFC, I know you love whitespace so added extra...

Good spot

File: src/TransformCore/PHE/HayApi/NarrativeBundle/Narrative/Results/ResultsNarrativeBuilder.php (hunk 1 above)
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/NarrativeBundle/Narrative/Results/ResultsNarrativeBuilder.php]: > I am probably overthinking this

Could be

**Pull request merged**